src/components/cards/OverlayCard.tsx

```diff
@@ -0,0 +1,205 @@
+import React from "react";
+import sanitizeHtml from "sanitize-html";
+import { palette } from "@guardian/src-foundations";
+import { FontCSS, TdCSS, TableCSS, ImageCSS } from "../../css";
+import { sanitizeOptions } from "../../styles/sanitize-options";
+import { Content } from "../../api";
+import { formatImage } from "../../image";
+import { kickerText } from "../../kicker";
+
+const fontSizes = {
+    large: {
+        fontSize: "22px",
+        lineHeight: "26px"
+    },
+    small: {
+        fontSize: "16px",
+        lineHeight: "20px"
+    }
+};
+
+const imgStyle: ImageCSS = {
+    outline: "none",
+    textDecoration: "none",
+    maxWidth: "100%",
+    clear: "both",
+    display: "block",
+    border: "0",
+    width: "100%",
+    fontFamily: "Georgia, serif",
+    color: palette.culture.main
+};
+
+const tableStyle: TableCSS = {
+    borderSpacing: 0,
+    borderCollapse: "collapse",
+    width: "100%"
+};
+
+const tdStyle = (backgroundColor: string): TdCSS => {
+    return {
+        padding: "0",
+        backgroundColor: backgroundColor || "transparent"
+    };
+};
+
+const headlineCellStyle = {
+    width: "93%",
+    backgroundColor: palette.neutral[7],
+    padding: "3px 40px 20px 10px"
+};
+
+const blankCellStyle = {
+    width: "7%"
+};
+
+const linkStyle: FontCSS = {
+    textDecoration: "none"
+};
+
+const headlineStyle = {
+    color: palette.neutral[100],
+    fontFamily: "'GH Guardian Headline', Georgia, serif",
+    fontWeight: 400,
+    ...fontSizes.large
+};
+
+const kickerStyle = {
+    color: palette.neutral[100],
+    fontFamily: "'GH Guardian Headline', Georgia, serif",
+    fontWeight: 700,
+    ...fontSizes.large
+};
+
+const trailTextStyle: FontCSS = {
+    fontFamily: "'GH Guardian Headline', Georgia, serif",
```

Comment:
Not something particularly related to this PR, but soon we should convert to using fonts from the design system - see `src/styles/typography.ts` here for what I mean.

Reply:
Agreed. I've added a Trello card so we can tackle this separately and clean up a lot of typography related code. Not sure what the approach will be yet but will look into options.
https://trello.com/c/U6P8tHxo/69-refactor-components-to-use-fonts-from-the-design-system